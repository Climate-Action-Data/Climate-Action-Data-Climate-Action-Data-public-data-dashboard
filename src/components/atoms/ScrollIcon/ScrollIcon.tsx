import { FC } from 'react'
import { Icon, IconProps } from '@chakra-ui/react'

export const ScrollIcon: FC<IconProps> = (props: IconProps) => {
  const actualProps = {
    ...props,
    width: props?.width ?? `12px`,
    height: props?.height ?? `13px`,
  }
  return (
    <Icon viewBox="0 0 12 13" {...actualProps}>
      <path
        fill="currentColor"
        d="M5.99953 12.6884L0.345703 7.0346L1.39953 5.98078L5.99953 10.5654L10.5995 5.98078L11.6534 7.0346L5.99953 12.6884ZM5.99953 6.70765L0.345703 1.05383L1.39953 0L5.99953 4.58463L10.5995 0L11.6534 1.05383L5.99953 6.70765Z"
      />
    </Icon>
  )
}
