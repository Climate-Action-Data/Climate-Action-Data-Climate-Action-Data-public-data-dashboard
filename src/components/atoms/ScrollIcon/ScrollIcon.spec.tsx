import { render } from '@testing-library/react'
import { ScrollIcon } from './ScrollIcon'

it(`renders correctly`, () => {
  const { container } = render(<ScrollIcon />)
  expect(container).toMatchSnapshot()
})
