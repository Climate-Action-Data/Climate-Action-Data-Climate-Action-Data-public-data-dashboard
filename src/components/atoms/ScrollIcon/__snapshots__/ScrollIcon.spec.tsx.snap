// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <svg
    class="chakra-icon css-sxms86"
    focusable="false"
    viewBox="0 0 12 13"
  >
    <path
      d="M5.99953 12.6884L0.345703 7.0346L1.39953 5.98078L5.99953 10.5654L10.5995 5.98078L11.6534 7.0346L5.99953 12.6884ZM5.99953 6.70765L0.345703 1.05383L1.39953 0L5.99953 4.58463L10.5995 0L11.6534 1.05383L5.99953 6.70765Z"
      fill="currentColor"
    />
  </svg>
</div>
`;
